test/component-movable-test.js

```diff
@@ -22,6 +22,16 @@ describe('Moveable component', () => {
     viewInstance.remove();
   }
 
+  function mouseMoveTo(x, y) {
```

Comment:
I would suggest to change this name to something like `dragAndDropElement` and would pass element there explicitly. It will be more generic then and name will describe, what this function actually mock.